ci/do_ci.sh

```diff
@@ -12,8 +12,11 @@ if [[ "$1" == "format_pre" || "$1" == "fix_format" || "$1" == "check_format" ||
   build_setup_args="-nofetch"
 fi
 
-SRCDIR="${PWD}"
+# TODO(phlax): Clarify and/or integrate SRCDIR and ENVOY_SRCDIR
+export SRCDIR="${SRCDIR:-$PWD}"
```

Comment:
im tempted to think that this is the cause of the ci fail - just on the basis of what has changed - but i dont see an obv reason to support that